CMakeLists.txt

```diff
@@ -147,6 +147,30 @@ endif ()
 message ("-- checking dependency LibVala")
 set (found FALSE)
 if (NOT found)
+  message ("--  checking option libvala-0.30")
```

Comment:
Changing the CMakeLists is the wrong approach here, as the cmake code is completely generated by valama. Instead, you should add the dependency in the UI, build the project and then commit the project file and the resulting cmake changes.